pdoc/__main__.py

```diff
@@ -188,17 +188,20 @@ def cli(args: list[str] = None) -> None:
         )
         return
     else:
-        all_modules = extract.walk_specs(opts.modules)
-        with pdoc.web.DocServer(
-            (opts.host, opts.port),
-            all_modules,
-        ) as httpd:
-            url = f"http://{opts.host}:{opts.port}"
+        try:
+            try:
+                httpd = pdoc.web.DocServer((opts.host, opts.port or 8080), opts.modules)
```

Comment:
that should be `port >= 1000` or otherwise linux is going to say no-no

Reply:
Not sure if I'm following here. It's whatever the user specified or 8080 by default. If you explicitly specify something <1024 on Linux then yes, you will get an error unless you have the required extra permissions.